### ISSUE TITLE
Elevated urine L-alloisoleucine concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6043]
**Preferred term label:** Elevated urine L-alloisoleucine concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of L-alloisoleucine in the urine is above the normal range., PMID: 7413299

**Parent term (use hpo.jax.org/app)** Isoleucinuria HP:0032371

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added as HP:0033156
